src/chat/chatter.py

```diff
@@ -298,6 +307,14 @@ def updateGame(self):
             elif game.state == GameState.PLAYING:
                 self.statusItem.setIcon(util.THEME.icon("chat/status/playing.png"))
                 self.statusItem.setToolTip("Playing Game<br/>"+url.toString())
+                mapname = game.mapname
+                icon = maps.preview(mapname)
+                if not icon:
+                    self.chat_widget.client.downloader.downloadMapPreview(mapname, self)
```

Comment:
The object you pass here will have a setIcon method called on it once the download is complete. You want to pass mapItem here, else the map will show in place of the flag.